### PR TITLE
refactor: remove FrontEndExample

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

Part of https://github.com/Tinkoff/taiga-ui/issues/1764

### CODE REVIEW

File: projects/demo/src/modules/charts/line-days-chart/line-days-chart.component.ts [hunk 1]
```diff
@@ -26,20 +15,20 @@ import {FrontEndExample} from '../../interfaces/front-end-example';
...
-    readonly example2: Record<string, string> = {
-        TypeScript: example2Ts,
-        HTML: example2Html,
-        LESS: example2Less,
-        Pipe: example2Pipe,
+    readonly example2: TuiDocExample = {
+        TypeScript: import('!!raw-loader!./examples/2/index.ts'),
+        HTML: import('!!raw-loader!./examples/2/index.html'),
+        LESS: import('!!raw-loader!./examples/2/index.less'),
+        Pipe: import('!!raw-loader!./examples/2/pipe.ts'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/charts/line-days-chart/line-days-chart.component.ts:31]:
```suggestion
       'pipe.ts': import('!!raw-loader!./examples/2/pipe.ts'),
```
(stackblitz)

File: projects/demo/src/modules/charts/pie-chart/pie-chart.component.ts [hunk 2]
```diff
@@ -1,41 +1,31 @@
...
     selector: 'example-tui-pie-chart',
     templateUrl: './pie-chart.template.html',
     styleUrls: ['./pie-chart.style.less'],
     changeDetection,
 })
 export class ExampleTuiPieChartComponent {
-    readonly exampleModule = exampleModule;
-    readonly exampleHtml = exampleHtml;
+    readonly exampleModule = import('!!raw-loader!./examples/import/import-module.txt');
+    readonly exampleHtml = import('!!raw-loader!./examples/import/insert-template.txt');
```
Comment (nsbarsukov) [inline projects/demo/src/modules/charts/pie-chart/pie-chart.component.ts:17]: `*.md` ?

File: projects/demo/src/modules/components/svg/svg.component.ts [hunk 3]
```diff
@@ -1,38 +1,30 @@
...
 @Component({
     selector: 'example-tui-svg',
     templateUrl: './svg.template.html',
     changeDetection,
 })
 export class ExampleTuiSvgComponent {
-    readonly exampleModule = exampleModule;
-    readonly exampleHtml = exampleHtml;
+    readonly exampleModule = import('!!raw-loader!./examples/import/import-module.txt');
+    readonly exampleHtml = import('!!raw-loader!./examples/import/insert-template.txt');
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/svg/svg.component.ts:16]: md ?

File: projects/demo/src/modules/tables/reorder/examples/import/import-module.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+```ts
+import {TuiReorderModule} from '@taiga-ui/addon-table/components/reorder';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/tables/reorder/examples/import/import-module.md:2]:
```suggestion
import {TuiReorderModule} from '@taiga-ui/addon-table';
```

File: projects/demo/src/modules/tables/reorder/reorder.component.ts [hunk 5]
```diff
@@ -1,24 +1,18 @@
...
     templateUrl: './reorder.template.html',
     changeDetection,
 })
 export class ExampleTuiReorderComponent {
-    readonly example1: FrontEndExample = {
-        TypeScript: example1Ts,
-        HTML: example1Html,
-    };
+    readonly exampleModule = import('!!raw-loader!./examples/import/import-module.txt');
+    readonly exampleHtml = import('!!raw-loader!./examples/import/insert-template.txt');
```
Comment (nsbarsukov) [inline projects/demo/src/modules/tables/reorder/reorder.component.ts:12]: md ?

File: projects/demo/src/modules/tables/resizable-column/examples/import/import-module.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+```ts
+import {TuiResizableColumnModule} from '@taiga-ui/addon-table/components/resizableColumn';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/tables/resizable-column/examples/import/import-module.md:2]:
```suggestion
import {TuiResizableColumnModule} from '@taiga-ui/addon-table';
```

File: projects/demo/src/modules/charts/line-days-chart/line-days-chart.component.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/charts/line-days-chart/line-days-chart.component.ts]: fixed

File: projects/demo/src/modules/charts/pie-chart/pie-chart.component.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/charts/pie-chart/pie-chart.component.ts]: fixed

File: projects/demo/src/modules/components/svg/svg.component.ts (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/components/svg/svg.component.ts]: fixed

File: projects/demo/src/modules/tables/reorder/examples/import/import-module.md (hunk 4 above)
Comment (splincode) [reply projects/demo/src/modules/tables/reorder/examples/import/import-module.md]: fixed

File: projects/demo/src/modules/tables/reorder/reorder.component.ts (hunk 5 above)
Comment (splincode) [reply projects/demo/src/modules/tables/reorder/reorder.component.ts]: fixed

File: projects/demo/src/modules/tables/resizable-column/examples/import/import-module.md (hunk 6 above)
Comment (splincode) [reply projects/demo/src/modules/tables/resizable-column/examples/import/import-module.md]: fixed

**Review by nsbarsukov — COMMENTED** [review]
File

https://github.com/Tinkoff/taiga-ui/blob/splincode/chore/demo/frontend-example/projects/demo/src/modules/charts/bar-chart/examples/import/insert-template.md

does not have code block

File: projects/demo/src/modules/services/table-bar/examples/import/service-usage-html.md [hunk 7]
```diff
@@ -0,0 +1 @@
+<ng-template #tableBarTemplate> <p>Template</p> </ng-template>
```
Comment (nsbarsukov) [inline projects/demo/src/modules/services/table-bar/examples/import/service-usage-html.md:1]: add code block

**Review by nsbarsukov — APPROVED** [review]
🔥

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
Cypress tests logs:
```
1) Demo
        components/accordion:
      AssertionError: Timed out retrying after 10000ms:
      Expected to find element: `.t-example`, but never found it.
      Queried from element: <tui-doc-example#single.ng-star-inserted>
```

File: projects/demo/src/modules/services/table-bar/examples/import/service-usage-html.md (hunk 7 above)
Comment (splincode) [reply projects/demo/src/modules/services/table-bar/examples/import/service-usage-html.md]: fixed

**Comment by splincode** [comment]
@nsbarsukov fixed

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/checkbox-block/examples/import/insert-template.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+```html
+<div [formGroup]="testForm">
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/checkbox-block/examples/import/insert-template.md:2]: nit:
```suggestion
<form [formGroup]="testForm">
```

File: projects/demo/src/modules/components/checkbox-labeled/examples/import/insert-template.md [hunk 9]
```diff
@@ -0,0 +1,5 @@
+```html
+<div [formGroup]="testForm">
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/checkbox-labeled/examples/import/insert-template.md:2]: nit:
```suggestion
<form [formGroup]="testForm">
```

File: projects/demo/src/modules/components/input-phone-international/examples/import/import-module.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+import {FormsModule, ReactiveFormsModule} from '@angular/forms'; import {TuiInputPhoneInternationalModule} from
+'@taiga-ui/kit';
+
+...
+
+@NgModule({ imports: [ ... FormsModule, ReactiveFormsModule, TuiInputPhoneInternationalModule ], ...
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-phone-international/examples/import/import-module.md:6]: add code block

File: projects/demo/src/modules/components/marker-icon/examples/import/import-module.md [hunk 11]
```diff
@@ -0,0 +1,5 @@
+import {TuiMarkerIconModule} from '@taiga-ui/kit';
+
+...
+
+@NgModule({ imports: [ ... TuiMarkerIconModule ], ...
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/marker-icon/examples/import/import-module.md:5]: add code block

File: projects/demo/src/modules/components/radio-block/examples/import/insert-template.md [hunk 12]
```diff
@@ -0,0 +1,22 @@
+```html
+<div [formGroup]="testForm">
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/radio-block/examples/import/insert-template.md:2]: nit:
```suggestion
<form [formGroup]="testForm">
```

and closing tag too

File: projects/demo/src/modules/components/radio-labeled/examples/import/insert-template.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
+```html
+<div [formGroup]="testForm">
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/radio-labeled/examples/import/insert-template.md:2]: nit:
```suggestion
<form [formGroup]="testForm">
```

File: projects/demo/src/modules/components/toggle/examples/import/insert-template.md [hunk 14]
```diff
@@ -0,0 +1,5 @@
+```html
+<div [formGroup]="testForm">
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/toggle/examples/import/insert-template.md:2]: nit:
```suggestion
<form [formGroup]="testForm">
```

**Review by nsbarsukov — CHANGES_REQUESTED** [review]
+ `projects/demo/src/modules/charts/bar-chart/examples/import/insert-template.md`

**Pull request merged**